jsonschema-validation.xml

```diff
@@ -817,6 +819,25 @@
                     </t>
                 </section>
 
+                <section title="uritemplate">
+                    <t>
+                        This attribute applies to string instances.
+                    </t>
+                    <t>
+                        A string instance is valid against this attribute if it is a valid URI Template
+                        (of any level), according to <xref target="RFC6570"/>.
+                    </t>
+                </section>
+
+                <section title="jsonpointer">
+                    <t>
+                        This attribute applies to string instances.
+                    </t>
+                    <t>
+                        A string instance is valid against this attribute if it is a valid JSON Pointer,
+                        according to <xref target="RFC6901"/>
```

Comment:
At this point it would be necessary to clarify if the JSON Pointer should be in its [string representation](https://tools.ietf.org/html/rfc6901#section-5) or [URI fragment identifier representation](https://tools.ietf.org/html/rfc6901#section-6) or is both permitted. This problem came up in #141